### ISSUE TITLE
Acessibility violation with table 

### ISSUE THREAD

**Issue by lavenal** [OpenLiberty/guide-docker#4]
The violation is reported as openliberty.io issue: https://github.com/OpenLiberty/openliberty.io/issues/230

After further investigation, the problem is with the table itself. To fix the accessibility violation, Add options="header" to the table attribute list. There are two tables in this guide and both have to change to add the options attribute.

[cols="70, 100", options="header"]
[cols="15, 100", options="header"]
